certified-connectors/Meisterplan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7710 @@
+          },
+          {
+            "name": "scenarioId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "0b9e0e29-8d87-4db8-baa8-1b1a49e354da",
+            "x-ms-summary": "ScenarioId",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProgramResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "409": {
+            "description": "Conflict"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          }
+        },
+        "security": [
+          {
+            "API-Token": []
+          },
+          {
+            "OAuth": []
+          }
+        ]
+      }
+    },
+    "/v1/scenarios/{scenarioId}/projects": {
+      "get": {
+        "tags": [
+          "Projects"
+        ],
+        "summary": "Get a List of Projects",
+        "description": "This endpoint is paginated. It returns a list of projects up to the specified PAGESIZE",
+        "operationId": "GetAllProjects",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "filter",
+            "in": "query",
+            "description": "Stringified JSON object with a subset of properties of the expected object. Selects all objects that exactly match the specified properties. Projects are searchable by projectKey, name, externalId, programId, portfolioId, and custom fields of the String and Integer type. Integers must be encoded as JSON string. The filter programId = null will yield all projects that are not associated with a program. Furthermore, the filter portfolioId = null will yield all projects. Note: take care to URL encode your parameter. { \"projectKey\": string, \"name\": string, \"externalId\": string, \"programId\": string, \"portfolioId\": string, \"customFields\": { \"cust_field_1\": string, \"cust_field_2\": string, .... } }",
+            "required": false,
+            "type": "string",
+            "x-example": "{\"customFields\": {\"cust_prj_number\": \"PR001\"}}",
+            "x-ms-summary": "Filter"
+          },
+          {
+            "name": "pageAfter",
+            "in": "query",
+            "description": "The cursor after which to retrieve results. The cursor can be taken from a preceding request.",
+            "required": false,
+            "type": "string",
+            "x-example": "ZTFlYzhhNTItY2Y4MC00NDYwLWI3MjEtYjFiMzM3YWYwYWUx",
```

Comment:
@itd-sk this one too looks like a secret-look-a-like-value, please remove or replace it.

Reply:
No secret-look-a-like-values should be present in the JSON file any more.